File: Difficulty: Easy/Second Largest/second-largest.py
class Solution:
    def getSecondLargest(self, arr):
        if len(arr) < 2:
            return -1
            
        max = arr[0]
        for i in range(1, len(arr)):
            if(arr[i] > max):
                max = arr[i]
        
        secondmax = -1
        for i in range(0, len(arr)):
            if( secondmax < arr[i] and arr[i] != max):
                secondmax = arr[i]
        return secondmax
                

File: Difficulty: Easy/Second Largest/second-largest.cpp
class Solution {
  public:
    int getSecondLargest(vector<int> &arr) {
        // code here
        int largest = INT_MIN;
        for(int i = 0; i < arr.size(); i++){
            if(arr[i] > largest){
                largest = arr[i];
            }
        }
        
        int secondLargest = INT_MIN;
        for(int i = 0; i < arr.size(); i++){
            if(arr[i] > secondLargest && arr[i] != largest){
                secondLargest = max(arr[i], secondLargest);
            }
        }
        
        if(secondLargest == INT_MIN) return -1;
        return secondLargest;
    }
};